lstchain/reco/r0_to_dl1.py

```diff
@@ -104,17 +104,21 @@ def get_dl1(
 
     config = replace_config(standard_config, custom_config)
     cleaning_parameters = config["tailcut"]
+    cleaning_parameters_for_tailcuts = cleaning_parameters.copy()
```

Comment:
I don't understand these changes, the code was good before. Doing a full copy and then later popping the values and then never using them is not an improvement.

Reply:
no, the code was not good if it was not working

Reply:
`cleaning_parameters` need to contain all dictionary entries (`use_main_island` and `delta_time` included) because the configuration is not reloaded every event that is analyzed.

If we pass `cleaning_parameters` to `tailcut_clean`, an error is raised because it does not identify some of the dictionary entries, that is why a new dictionary without the `use_main_island` and `delta_time` entries are included is created every time.

Reply:
I see, then the error is that this is in the event loop in the first place. It does not change.